setup.py

```diff
@@ -36,7 +36,7 @@
     license="Apache License 2.0",
     install_requires=[
         "absl-py",
-        "numpy",
+        "numpy~=1.23.2",  # Numpy 1.24 breaks ragged tensors. Pin to keras build
```

Comment:
Hmm, not sure about this? Maybe let's leave this off for now.

This would mandate a newer version of numpy than colab has by default right now. And if people did have a reason to use 1.24, we are prohibiting them for what appears to be only a test breakage.

We've generally try to go strict for requirements, loose for setup.py, as describer here. https://stackoverflow.com/a/33685899

Reply:
Done.  Left setup.py as-is for numpy version.